src/modules/billing/jobs/prepare-transactions.js

```diff
@@ -9,14 +10,19 @@ const { BATCH_ERROR_CODE } = require('../../../lib/models/batch');
 const batchJob = require('./lib/batch-job');
 const helpers = require('./lib/helpers');
 const { jobName: createChargeJobName } = require('./create-charge');
+const { jobName: refreshTotalsJobName } = require('./refresh-totals');
+
 const { logger } = require('../../../logger');
 const supplementaryBillingService = require('../services/supplementary-billing-service');
 const billingTransactionsRepo = require('../../../lib/connectors/repos/billing-transactions');
+const Transaction = require('../../../lib/models/transaction');
 
 const createMessage = partial(helpers.createMessage, JOB_NAME);
 
 const getTransactionId = transaction => transaction.billingTransactionId;
 
+const isCandidateTransaction = transaction => transaction.status === Transaction.statuses.candidate;
```

Comment:
Would you kindly slot in an inline comment explaining what a candidate transaction is please?

Reply:
Added to the service model